scripts/run-e2e-config.ts

```diff
@@ -144,6 +143,16 @@ export const cra: Parameters = {
   ].join(' && '),
 };
 
+export const cra3: Parameters = {
+  name: 'cra',
+  version: '3.4.1',
+  generator: [
+    'npx create-react-app@{{version}} {{name}}-{{version}}',
```

Comment:
The results of this (at least package.json) looks identical to what I'd get running the latest version (uses react-scripts@4 and react@17). Can we make this better? @tooppaaa

Reply:
Extracted from repository now